recipes/cern-root/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "v6-22-02":
```

Comment:
Does it follow semver?

If you put this `v` at the beginning, it will have consequences on `package_id` of recipes depending on `cern-root`.

Reply:
This is a complicated question and I am not sure what is the correct solution.

ROOT does not follow strict semantic versioning. They describe their versioning system at: <https://root.cern/about/versioning/>.

In summary: 
- They may introduce breaking changes in minor releases. 
- New minor versions of ROOT are guaranteed to read files generated by old versions but old versions are not guaranteed to read files generated with newer versions.
- ROOT does not guarantee ABI backward compatibility across versions.

So, in general, it is not guaranteed to be safe to swap in a newer minor release for an older one.

Reading the Conan documentation about [package_id and ABI compatibility](https://docs.conan.io/en/latest/creating_packages/define_abi_compatibility.html) this makes me think that that we do not want to make this change. However, I could be wrong. Someone who has a better understanding of Conan than me may correct me.

Reply:
Reading the documentation for cern I'd argue it should be `6.2/02` to match the official syntax

> ROOT versioning scheme
The naming convention for ROOT versions is v<MAJOR>.<MINOR>/<PATCH> (e.g. v6.20/04).

With or without the `v` it's breaks semver and that's fine

Reply:
The `vMAJOR-MINOR-PATCH` convention is what ROOT use in its branch names.

But I think that you a right. It would be better to follow the `vMAJOR.MINOR/PATCH` convention that is used in the documentation. I will make that change.

Reply:
Hmm. When I try to switch the name to `vMAJOR.MINOR/PATCH` I get errors:
```
ERROR: Error parsing lockfile '/tmp/tmpbuild/build/conan.lock': The reference has too many '/'
```
I guess that something in Conan is assuming that version numbers won't contain a "/" character.

The closest that I can get, with this limitation, to the ROOT version naming convention is: `vMAJOR.MINOR.PATCH`.

Reply:
Sounds like a bug 🐛 

What you are currently using matches upstream which it better than lying about being semver.

Thanks for trying!